cmd/kubeadm/app/phases/copycerts/copycerts.go

```diff
@@ -234,6 +234,8 @@ func DownloadCerts(client clientset.Interface, cfg *kubeadmapi.InitConfiguration
 		return errors.Wrap(err, "error decoding secret data with provided key")
 	}
 
+	fmt.Printf("[download-certs] Saving the certificates to the folder: %q\n", cfg.CertificatesDir)
```

Comment:
```suggestion
	fmt.Printf("[download-certs] Download the certificates to the %q directory\n", cfg.CertificatesDir)
```
Or `s/Saving/Save/g`.

Reply:
let's use "folder" instead,  as I found this word is preferred in this project: https://github.com/kubernetes/kubernetes/blob/9b4b1c0e79aca46965137227e2c9cde0d285cabb/cmd/kubeadm/app/cmd/phases/join/controlplaneprepare.go#L193